### ISSUE TITLE
Sudden increment in loss values during training

### ISSUE THREAD

**Issue by purvang3** [xiaohu2015/SwinT_detectron2#19]
<img width="1785" alt="Screen Shot 2022-03-03 at 6 13 53 PM" src="https://user-images.githubusercontent.com/39809304/156704071-e48e0051-acdd-4ca0-a376-c5bafdacc2c4.png">

**Comment by xiaohu2015** [comment]
@purvang3 do you test your dataset by other detection models?

**Comment by purvang3** [comment]
Thank you for the reply. I tried same dataset with YOLOX model and it is training well.

**Comment by purvang3** [comment]
Thank you for the reply. It was dataset only. Could you point out to the document and responsible script, which periodically evaluate saved checkpoint on validation data?

**Comment by xiaohu2015** [comment]
@purvang3 you can reference detectron2 document. For the checkpoint, you can set https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py#L541
for eval, you can set https://github.com/facebookresearch/detectron2/blob/main/detectron2/config/defaults.py#L592

**Comment by purvang3** [comment]
@xiaohu2015 Thank you for the reply and pointing out to the doc regarding eval. I am able to run periodic eval but coco metric validation result show either garbage values or not improving mAP but I do see decrement in training loss. Below I have attached image. Do you have any suggestion? let me know if you need any additional information.

<img width="1792" alt="Screen Shot 2022-03-09 at 2 09 41 PM" src="https://user-images.githubusercontent.com/39809304/157544927-bdc7d953-879a-4ca1-a582-ad84d8b12867.png">

<img width="1789" alt="Screen Shot 2022-03-09 at 2 09 02 PM" src="https://user-images.githubusercontent.com/39809304/157544940-17bde5d7-7d72-4376-ac2c-b9d2aada6342.png">

**Comment by xiaohu2015** [comment]
check your dataset, use https://github.com/facebookresearch/detectron2/blob/main/tools/visualize_data.py to show dataloader to check if your data is good.
